scons/plan.sh

```diff
@@ -0,0 +1,25 @@
+pkg_name=scons
+pkg_distname=$pkg_name
+pkg_origin=core
+pkg_version=2.5.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('mit')
```

Comment:
Please use the the format shown in the [SPDX License List](http://spdx.org/licenses/).

Also a `pkg_description` would be good.